docs/resource-policy/policy/balloons.md

```diff
@@ -188,6 +188,10 @@ Balloons policy parameters:
   - `cpuClass` specifies the name of the CPU class according to which
     CPUs of balloons are configured. Class properties are defined in
     separate `cpu.classes` objects, see below.
+  - `rdtClass` specifies the name of the RDT class. Corresponding
```

Comment:
nit: whether a non-existent class will result in failure to create the container depends (unfortunately) on the runtime in use and its configuration. Notably in containerd the default configuration implies failure, but this can be changed by flipping a configuration variable.